tiledb/sm/config/config.cc

```diff
@@ -606,6 +606,8 @@ Status Config::sanity_check(
     RETURN_NOT_OK(utils::parse::convert(value, &vuint64));
   } else if (param == "sm.memory_budget_var") {
     RETURN_NOT_OK(utils::parse::convert(value, &vuint64));
+  } else if (param == "sm.memory_budget_validity") {
```

Comment:
Does this need to be removed as well?

Reply:
Yes -- thanks. Not sure how I missed this.

Reply:
Done